### DISCUSSION TITLE
Commodore 64 and Amiga image formats extraction support

### DISCUSSION THREAD

**Discussion by eadmaster** [claunia/DiscImageChef#849] [Q&A]
I've found some references for these in the code, but they are not extracted:

https://github.com/aaru-dps/Aaru/blob/devel/Aaru.Decoders/Floppy/Commodore.cs
https://github.com/aaru-dps/Aaru/blob/devel/Aaru.Decoders/Floppy/Amiga.cs

e.g.
````
$ aaru -v filesystem extract jp2.adf jp2

aaru 5.3.1
Copyright © 2011-2022 Natalia Portillo

Image format identified by Raw Disk Image (12345678-aaaa-bbbb-cccc-123456789000).
Raw disk image contains a disk of type CBM_AMIGA_35_DD
1 partitions found.

Partition 1:
Identifying filesystem on partition

````

(nothing is written in output dir)

**Comment by robin-francois** [comment]
Hello,
If I am not mistaken, those filesystems can be identified, but files cannot be listed or extracted.

As stated here: https://github.com/aaru-dps/Aaru/blob/devel/README.md
Those formats are only for identification and information.
